### DISCUSSION TITLE
Update to 2.5.2 not possible

### DISCUSSION THREAD

**Discussion by n17r0l** [brutella/hkknx-public#262] [Q&A] [answered]
Dear all,

I'm currently getting errors when trying to update to 2.5.2 from the web interface:

Das Update auf Version 2.5.2 ist fehlgeschlagen..
File '.hkknx.old' for the command is not found in https://github.com/brutella/hkknx-public/releases/download/2.5.2/hkknx-2.5.2_linux_arm.tar.gz

Anyone has a suggestion to solve it?

Thannks & best regards

**Comment by brutella** [comment]
Which version of hkknx are you currently using?

**Comment by n17r0l** [comment]
I'm using 2.5.0 and the update to 2.5.1 also failed. I have installed the preprepared pi-image, using it since 2 years in trail without issued and recently updated to the licenced version.

**Reply by brutella** [reply, marked as answer]
Please do restart hkknx (Settings → More → Restart System) and then try the update again.

**Reply by n17r0l** [reply]
It worked - thanks!

**Reply by xchillsx** [reply]
Same thing here - worked now!

A W E S O M E 👍
